pixel-saver@deadalnix.me/schemas/org.gnome.shell.extensions.pixel-saver.gschema.xml

```diff
@@ -0,0 +1,9 @@
+<schemalist>
+	<schema id="org.gnome.shell.extensions.pixel-saver" path="/org/gnome/shell/extensions/pixel-saver/">
+		<key name="show-buttons" type="b">
```

Comment:
In #53 I'm adding boolean key "buttons-placement-left" to move buttons to the left next to the title. Maybe we should make a drop down with placement options "left", "right", "hidden" or something like that.